WordPress/Classes/Models/ReaderDefaultTopic.swift

```diff
@@ -4,4 +4,26 @@ import Foundation
     override open class var TopicType: String {
         return "default"
     }
+
+    /// - note: Starting with 25.5, the app no longer uses the default topics
+    /// fetched from `/read/menu`. Instead, it uses the topics created locally.
+    /// The only reason these topics are needed is to filter entities in Core Data.
+    static func make(path: Path, in context: NSManagedObjectContext) throws -> ReaderDefaultTopic {
+        if let topic = context.firstObject(ofType: ReaderDefaultTopic.self, matching: NSPredicate(format: "path == %@", path.rawValue)) {
+            return topic
+        }
+        let topic = ReaderDefaultTopic(entity: ReaderDefaultTopic.entity(), insertInto: context)
+        topic.type = TopicType
+        topic.path = path.rawValue
+        topic.title = ""
+        try context.save()
```

Comment:
We should use `performAndSave` instead of saving the context directly. Or, at bare minimal, use `ContextManager.save`.

Also, we don't update and save the main context.

Reply:
> Also, we don't update and save the main context.

That's not exactly the case as a large number of features use save on main, as I think they should. I'd suggest phasing out the background context in the future.